docs/src/pages/[platform]/connected-components/authenticator/customization/customization.full-ui-customization.flutter.mdx

```diff
@@ -12,91 +12,151 @@ Below is an example of how you can use `authenticatorBuilder` to add a logo to b
 This example uses several prebuilt widgets from the amplify_authenticator package, such as `SignInForm()`, `SignUpForm()`, `SignInFormField.username()`, and `SignInButton()`. All of the prebuilt widgets are integrated into the authenticator's state, meaning that you do not have to add an `onChanged` callback for the Form Fields, or an `onTap` callback for the Buttons. You can find a full list of the prebuilt forms, form fields, and buttons in <Link isExternal href="https://pub.dev/documentation/amplify_authenticator/latest/amplify_authenticator/amplify_authenticator-library.html">the API docs</Link>.
 
 ```dart
```

Comment:
The code block is pretty long, I wonder if we should move it to an example file

Reply:
Updated